homeassistant/components/tuneblade_remote/__init__.py

```diff
@@ -0,0 +1,59 @@
+"""Integration to integrate TuneBlade Remote devices with Home Assistant."""
+
+from datetime import timedelta
+import logging
+
+from pytuneblade import TuneBladeApiClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import DOMAIN
+from .coordinator import TuneBladeDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(seconds=5)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
we should extend the ConfigEntry type to type the runtime data, please check `runtime_data` rule

Reply:
Please extend the Configentry type so the type checker also knows that it's typed. Check the docs for the runtime_data rule :)